src/Ormolu/Printer/Internal.hs

```diff
@@ -326,12 +327,17 @@ setIndent i m' = do
   R (local modRC m)
   traceR "set_indent_after" Nothing
 
--- | Get 'RealSrcSpan' of enclosing span for given referencne span.
+-- | Get 'RealSrcSpan' of enclosing span for given reference span.
 
 getEnclosingSpan :: RealSrcSpan -> R (Maybe RealSrcSpan)
 getEnclosingSpan r =
   listToMaybe . dropWhile (== r) <$> R (asks rcEnclosingSpans)
 
+-- | Get 'RealSrcSpan' of the current enclosing span.
+
+getEnclosingSpan' :: R (Maybe RealSrcSpan)
```

Comment:
When situations like this arise, it's often better to make the old function more powerful instead of adding more versions of it.